### DISCUSSION TITLE
Is it possible to use Zluda on GROMACS?

### DISCUSSION THREAD

**Discussion by TuashDxy** [vosen/ZLUDA#301] [Q&A]
I have some project with gromacs, since it was a CUDA based and i'm currently use AMD. Is it possible to use zluda with GROMACS? FYI gromacs can only be run on linux...

**Comment by vosen** [comment]
With the old ZLUDA I didn't try GROMACS, because supposedly it's going to be SYCL-only and support all GPUs equally.
New ZLUDA is not at the point where it can run anything at all.
Are you using GROMACS as a library?

**Reply by TuashDxy** [reply]
yes,  GPU-accelerated libraries for MD (molecular dynamics) run

**Reply by TuashDxy** [reply]
i didnt understand how to use other alternative such us SYCL, OPENCL. etc in gromacs. Is it actually work the same as cuda toolkit does?
